WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelAddNewPackage.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+struct ShippingLabelAddNewPackage: View {
+    @Environment(\.presentationMode) var presentation
+    @ObservedObject private var viewModel = ShippingLabelAddNewPackageViewModel()
+
+    var body: some View {
+        VStack(spacing: 0) {
+            VStack(spacing: 0) {
+                SegmentedView(selection: $viewModel.selectedIndex, views: [Text(Localization.customPackage), Text(Localization.servicePackage)])
+                    .frame(height: 44)
+                Divider()
+            }
+            ScrollView {
+            }
+        }
+        .navigationTitle(Localization.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden(true)
+        .toolbar {
+            ToolbarItem(placement: .navigationBarLeading) {
+                Button {
+                    presentation.wrappedValue.dismiss()
+                } label: {
+                    Image(uiImage: .chevronImage.withHorizontallyFlippedOrientation().imageFlippedForRightToLeftLayoutDirection())
+                }
+
+            }
+        }
```

Comment:
SwiftUI doesn't seem to have an equivalent to UIKit's `navigationItem.backButtonDisplayMode = .minimal` for removing text from the navigation bar back button. Instead, we have to remove the default back button and then display our own custom back button.

We should make this reusable for any SwiftUI views that need it. I decided to wait on that for now, since I'm not sure if it would be more useful to have a reusable `ToolbarItem` for the minimal back button or a custom modifier that does all the toolbar setup for this.

Reply:
I'd say that a custom modifier would be great for reusability here! But I think we can use `.navigationBarItems(leading:)` here instead of `toolbar` as it's more straightforward that we're dealing with navigation bar here - plus it's shorter! What do you think?

Also - thank you for adding `imageFlippedForRightToLeftLayoutDirection()` for RTL languages here 🎉. Do you think it's necessary to add `.chevronImage.withHorizontallyFlippedOrientation()` to the `UIImage+Woo` extension as something like `chevronLeftImage`? Just in case it may be necessary in other places as well.

Reply:
> I think we can use .navigationBarItems(leading:) here

I used that originally, but then I [looked at the docs](https://developer.apple.com/documentation/swiftui/view/navigationbaritems(leading:trailing:)) and realized it's deprecated. 😬 The recommended approach is `toolbar(content:)` now, which is why I used it here.

> Do you think it's necessary to add .chevronImage.withHorizontallyFlippedOrientation() to the UIImage+Woo extension as something like chevronLeftImage?

Good point, I'll go ahead and add that! I just realized there's a Gridicon for the left chevron so we don't have to flip the regular chevron for that, either.

Reply:
I'm going to go ahead and merge this PR as-is, and I'll make both changes (the custom modifier for reusability, and the left-facing chevron image) in a followup PR.